Rakefile

```diff
@@ -0,0 +1,89 @@
+require 'json'
+require 'fileutils'
+require 'mkmf'
+require 'aws-sdk'
+require 'logger'
+
+include FileUtils
+
+task :version do
+ @version = JSON.parse(File.read('package.json'))['version']
+end
+
+task :base_dir do
+  @base_dir = File.dirname(File.expand_path(__FILE__))
+end
+
+task :install do
+  sh 'npm install --only=prod'
+end
+
+task :shrinkwrap => [:install] do
+  sh 'npm shrinkwrap'
+end
+
+task :pack => [:shrinkwrap] do
+  sh 'npm pack'
+end
+
+desc 'Release props.d and prepare to create a release on github.com'
```

Comment:
Seeing `props.d` reminds me that we should go through and standardize the naming across the entire project. I'm going to open an issue to address this, but in the meantime, can you change it to `propsd`?